plugins/net_plugin/net_plugin.cpp

```diff
@@ -261,13 +261,17 @@ namespace eosio {
       bool verify_catchup( const connection_ptr& c, uint32_t num, const block_id_type& id ); // locks mutex
 
    public:
+      enum class closing_mode {
+         immediately,
+         back_off
```

Comment:
This `closing_mode` is confusing to me. It is used to determine if you close or handshake. 
Not sure what `backoff` implies. It currently means: don't close but rather send a handshake.